packages/ember-data/lib/serializers/rest2-serializer.js

```diff
@@ -0,0 +1,138 @@
+import RESTSerializer from "ember-data/serializers/rest-serializer";
+import {
+  _normalizeSerializerPayloadItem,
+  pushPayload
+} from "ember-data/system/store/serializer-response";
+
+var forEach = Ember.ArrayPolyfills.forEach;
+//var map = Ember.ArrayPolyfills.map;
+
+function coerceId(id) {
+  return id == null ? null : id + '';
+}
+
+var REST2Serializer = RESTSerializer.extend({
+
+  normalize: function(typeClass, hash, prop) {
+    return {
+      data: _normalizeSerializerPayloadItem(typeClass, this._super(typeClass, hash, prop)),
+      included: []
+    };
+  },
+
+  normalizeResponse: function(store, primaryTypeClass, payload, id, requestType) {
+    var document = {
```

Comment:
Yeah, I'm not too fond of `document` but I'd like to have a more distinct name than `payload` or something like that. Any suggestions?